query/parse.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/grafana/regexp"
 )
 
+const RegexpFlags syntax.Flags = syntax.ClassNL | syntax.PerlX | syntax.UnicodeGroups
```

Comment:
Do we have to export this? Are we planning to use `RegexpFlags` in Sourcegraph?

Reply:
I think we copy pasted the regexp flags in sourcegraph and they actually ended up being different. So I think the intention is to directly use it. Although, now that you are exporting RegexpQuery that may no longer be needed?

Maybe having the sourcegraph PR accompany it would help us understand more context to review this?